src/map/directive.js

```diff
@@ -1,12 +1,22 @@
-goog.provide('ngeo.mapDirective');
+goog.provide('ngeo.map.directive');
 
 goog.require('goog.asserts');
 goog.require('ngeo');
+/** @suppress {extraRequire} */
 goog.require('ngeo.datasource.SyncDataSourcesMap');
 goog.require('ol.events');
 goog.require('ol.Map');
 
 
+/**
+ * @type {!angular.Module}
+ */
+ngeo.map.directive = angular.module('ngeoMap', []);
+
+const ngeoModule = ngeo.module;
```

Comment:
This only works for `goog.module` (because with `goog.provide` it pollutes the window object (with window.ngeoModule property)).

Reply:
So please use `ngeo.module` directly and disable the check (put it as warning).